client/src/views/ScorecardView.vue

```diff
@@ -2,7 +2,8 @@
   <div>
     <!-- Cover the entire page with loading element until data is ready. -->
     <div class='loading' v-if='!showScorecard'>
-      <loading :active='true' :is-full-page='false' color='#2553A0' loader='bars' opacity='1' />
+      <loading :active='true' :is-full-page='false' color='#2553A0'
```

Comment:
can we put each property on a newline like 
<loading :active='true'
                :is-full-page='false'
                :color='' 
.... 
or put back to single line instead of splitting down the middle

Reply:
Done!